config/config.yaml

```diff
@@ -2,3 +2,10 @@
 # In case of sample based data, it should be complemented by a samples.tsv file that contains
 # one row per sample. It can be parsed easily via pandas.
 pepfile: config/pep/config.yaml
+
+# minimal confidence of the LSTM network of the detected word
+min-confidence: 0.6
+
+# maximal number number of single-character edits that are required to transform 
+# one string (the word on the image) into another (the word in the personal data)
+max-distance: 2
```

Comment:
add newline at eof

Reply:
done in 0b5e45b. But why is this important?